client/src/index.css

```diff
@@ -11,3 +11,31 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.frame {
```

Comment:
since `index.css` is meant for globally used CSS, I'm wondering if it would be beneficial to split these into it's own stylesheet, unless you do think that `.frame` and `.image` will be used widely across the app

Reply:
Thank Bonnie for providing me the font-family. I will try to use this font-family globally.

Reply:
Also to bonnie's point - since this is global across the whole app, you should try to avoid using it unless you really need it - instead you should be using the React Hooks Style API with material-ui

Reply:
I totally agree with both of you. I use the React Hooks Style API with Material-UI first. However, I want to change the direction of Grid according to the width of the device and set the width of the image fit to mobile width but the height of the image fit to desktop height, so I use @media in CSS file.

Reply:
Use the React Hooks Style API with Material-UI instead of @media in index.css